### PR TITLE
Default to hide summaries and fix associated bugs

### LINKED ISSUE #1860: [VSLC] No option to talk about individual subjects
**Package**

VSLCV

**Is your feature request related to a problem? Please describe.**

Most Zooniverse projects have a "Done" or "Done and Talk" option, to automatically allow subjects to be ported to the talk portion of the project. I cannot find an option to "talk" about individual subjects.

**Describe the solution you'd like**
Having an option to connect subjects to the Talk boards would be useful.

**Describe alternatives you've considered**
I also like that the process is fairly streamlined for the project at the moment. So, adding a second click to complete a subject classification may not be beneficial overall. 
 
**Additional context**
It's possible there is a "switch" that needs to be toggled in the project builder that would add this functionality. If I missed that sorry.

### PR DESCRIPTION
**Pull request by srallen**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package: lib-classifier

Closes #1860

Describe your changes:
We've learned from PFE that the vast majority of projects wish the hide the classification summaries. This is because the summary doesn't really have much value add to the volunteer. It displays the JSON of the annotation values and that doesn't have a whole lot of meaning to them. We had decided that we'll reverse this logic for the new classifier and this implements it. The workflow configuration model now defaults to true for `hide_classification_summaries`. 

If a case can be made to have some kind of summary, then the lab editor will update to be able to set this config to false explicitly as well as have likely configurations for what the summary does. #1272 is the related issue.

Defaulting this to hide revealed a bug in setting the steps when steps are defined from the outset on a workflow rather than converted from the legacy setup from the project builder. This is now fixed and tested for. 

We now default to displaying Done & Talk button:

Kitty Zoo now shows a disabled Done & Talk until you pick an option (our baseline development project, load by default by the dev classifier https://localhost:8080/)
Screen Shot 2020-10-22 at 11 49 29 AM

Zwicky's Stellar Sleuths (https://localhost:8080/?project=4878&workflow=16135&env=production) now correctly shows Done and Done & Talk (they set the config in the interim, though so not a specific test anymore):
Screen Shot 2020-10-22 at 11 51 32 AM

PH TESS (https://localhost:8080/?project=7929&env=production) showing no regression:

Screen Shot 2020-10-22 at 11 53 10 AM

These can be seen by pulling down the repo, running `yarn bootstrap` in the root, then running the dev classifier using `yarn dev` in the lib-classifier folder.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

**Comment by srallen** [comment]
@ErikOstlund there's no rush on this one since Zwicky's Stellar Sleuths have explicitly set the configuration now for this. Happy to pair on the review on this if you like.

File: packages/lib-classifier/src/store/WorkflowStepStore.js [hunk 1]
```diff
@@ -151,8 +153,15 @@ const WorkflowStepStore = types
     }
 
     function setSteps (workflow) {
-      workflow.steps.forEach(([ stepKey, step ]) => {
-        self.steps.put(Object.assign({}, step, { stepKey }))
+      const { steps } = workflow
+      steps.forEach(([ stepKey, step ], index) => {
+        let next
+        if (steps[index + 1]) {
+          const nextStepKey = steps[index + 1][0]
```
Comment (eatyourgreens) [inline packages/lib-classifier/src/store/WorkflowStepStore.js:160]: This is hard to follow. Why isn't the next step key `steps[index + 1]['stepKey']`?

Comment (eatyourgreens) [reply packages/lib-classifier/src/store/WorkflowStepStore.js:160]: Maybe `const [ nextStepKey ] = steps[index + 1]` would be easier to read, if `steps` is an array of `[ stepKey, step]` pairs?

Comment (srallen) [reply packages/lib-classifier/src/store/WorkflowStepStore.js:160]: Steps aren't stored in a map in Panoptes DB because JSON doesn't support it, thus at this point in the code it's `[[stepKey, step]]`. ADR 10 is about this.

**Comment by srallen** [comment]
@eatyourgreens would you give this a final review?

**Comment by eatyourgreens** [comment]
Sure!

**Review by eatyourgreens — APPROVED** [review]
Looks good to me.
